cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsDownscaleService.java

```diff
@@ -67,17 +71,22 @@
                 instanceIds.add(vm.getInstanceId());
             }
 
+            AwsCredentialView credentialView = new AwsCredentialView(auth.getCloudCredential());
+            AuthenticatedContextView authenticatedContextView = new AuthenticatedContextView(auth);
+            String regionName = authenticatedContextView.getRegion();
+            awsCloudWatchService.deleteCloudWatchAlarmsForSystemFailures(stack, regionName, credentialView, instanceIds);
```

Comment:
`deleteCloudWatchAlarmsForSystemFailures()` should only perform actions for FreeIPA because only FreeIPA management service sets `CLOUDWATCH_CREATE_PARAMETER`.

```
        String isCreate = stack.getParameters().get(CLOUDWATCH_CREATE_PARAMETER);
        if (isCreate != null && isCreate.equals(Boolean.TRUE.toString())) {
```

Reply:
ok I missed this isCreate